kas.md

```diff
@@ -0,0 +1,441 @@
+# kas - 
```

Comment:
@macpijan maybe "kas: how to manage bitbake BSPs?" or "kas: how to manage Yocto and OE BSPs?"